data/mods/Xedra_Evolved/design_doc_spoilers.md

```diff
@@ -53,7 +53,29 @@ Principles of Blood Arts:
 
 ### Vampire Anathema
 
-TBD
+There exists a way to push vampirism beyond its limits.  This isn't something the mentors can teach, as almost none of them know about this method.  The few who know that this empowerment is possible keep it a complete secret, for its method is directly opposed to their plan of creating as many strong and willing vampires as possible.
+
+Once ago, a vampire hunter decided to fight fire with fire and sought a way to consume other vampires to gain the power to eradicate them all.  They succeeded, but won't ever share how they did to gain such power.  They relentlessly roam the world, searching for vampire-blooded to kill and drain.  Once the player reaches the final tier of vampirism, they too will eventually get a visit from the Anathema.
```

Comment:
```suggestion
Once years ago, a vampire hunter decided to fight fire with fire and sought a way to consume other vampires to gain the power to eradicate them all.  They succeeded, but won't ever share what they did to gain such power.  They relentlessly roam the world, searching for vampire-blooded to kill and drain.  Once the player reaches the final tier of vampirism, they too will eventually get a visit from the Anathema.
```

Reply:
Wouldn't "Years ago" fit better than "Once years ago"? 

However, from what I know, "once ago" is the most appropriate of the three to designate things that happened far in the past.

Reply:
Also if you want to add some design parameters on cryptids that’s cool or I can hit that when I get to adding some sections.

Reply:
Tomorrow I'll edit that part to be "Long ago".  I find it better than "Once long ago" and AFAIK it is proper english grammar.

I'll also rename that PR and include some cryptid advice too.  Their need to be unique, their falcultative loot, how to find them and that the cryptid hunter start has to get a random note and map for one cryptid at random from all the exisiting cryptids.

Is there anything else I should include too?

Reply:
I think that makes sense.  There's some additional things I'd like to eventually accomplish with cryptids that requires either some heavy npc lifting and/or c++.  But this should be everything needed for people to add new ones.